### ISSUE TITLE
ERR_INVALID_ARG_TYPE after upgrading node.js to v14

### ISSUE THREAD

**Issue by JoHuang** [mongodb-js/runner#177]
Node.js v14.5.0, mongodb-runner cannot get started
It worked on node v12.

```
> mongodb-runner start --topology=replicaset && npm run amendJest

  ◟ Starting a MongoDB deployment to test against...internal/fs/utils.js:701
    throw new ERR_INVALID_ARG_TYPE(
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received type number (50900)
    at writeFile (fs.js:1385:5)
    at go$writeFile (/Users/apple/MyProj/node_modules/graceful-fs/graceful-fs.js:139:14)
    at Object.writeFile (/Users/apple/MyProj/node_modules/graceful-fs/graceful-fs.js:136:12)
    at Object.writeFile (/Users/apple/MyProj/node_modules/universalify/index.js:5:67)
    at /Users/apple/MyProj/node_modules/mongodb-runner/lib/index.js:240:12
    at nextTask (/Users/apple/MyProj/node_modules/mongodb-runner/node_modules/async/dist/async.js:4576:27)
    at next (/Users/apple/MyProj/node_modules/mongodb-runner/node_modules/async/dist/async.js:4584:13)
    at /Users/apple/MyProj/node_modules/mongodb-runner/node_modules/async/dist/async.js:321:20
    at /Users/apple/MyProj/node_modules/mongodb-runner/lib/index.js:235:11
    at getPIDPath (/Users/apple/MyProj/node_modules/mongodb-runner/lib/index.js:22:3) {
  code: 'ERR_INVALID_ARG_TYPE'
}
npm ERR! Test failed.  See above for more details.
```

**Comment by dplewis** [comment]
Looks like this has been fixed. https://github.com/mongodb-js/runner/pull/174

Waiting for a release.

**Comment by dplewis** [comment]
@imlucas Thank you for working on this amazing project. Is there a chance for a release? Would love to use this on Node 14+

**Comment by j** [comment]
@addaleax @imlucas Any chance to get a release soon with Node 14+ working?

**Comment by j** [comment]
@dplewis @JoHuang You can get by for now using "github:mongodb-js/runner.git#dfb9a520147de6e2537f7c1c21a5e7005d1905f8" as your dependency.

**Comment by JoHuang** [comment]
Thanks. I'll try it when I got time.

**Comment by addaleax** [comment]
@rose-m just published `mongodb-runner@4.8.1`, I think this can be closed now :)
